src/zope/schema/_bootstrapfields.py

```diff
@@ -505,11 +508,13 @@ class Text(MinMaxLen, Field):
     """A field containing text used for human discourse."""
     _type = text_type
 
-    def __init__(self, *args, **kw):
+    def __init__(self, unicode_normalization='NFC', *args, **kw):
```

Comment:
I think it would be good to avoid changing the signature, and define `unicode_normalization` as akeyword-only argument, essentially.
```suggestion
    def __init__(self,  *args, **kw):
        self.unicode_normalization = kw.pop('unicode_normalization', 'NFC')
```

It might also be nice to make the default normalization value a class attribute. That way subclasses could more easily (declaratively) override it in a single place.

Reply:
@jamadden Thank you very much for your interest! And I'm totally agreeing on the signature! I will change that! The thing with the class attribute: I do think that in the rare cases where somebody would want to keep mixed unicode they'd not be happy to have that on every field (I'm supposing that this would happen if it's sticking with the class).

Reply:
>  I do think that in the rare cases where somebody would want to keep mixed unicode they'd not be happy to have that on every field (I'm supposing that this would happen if it's sticking with the class).

I'm sorry, I don't follow this. Could you elaborate some please?

I was probably unclear too. What I was suggesting was this:

```python
class Text(…):
   DEFAULT_UNICODE_NORM_FORM = 'NFC'

   def __init__(self, *args, **kwargs):
        self.unicode_normalization = kwargs.pop('unicode_normilazation', self.DEFAULT_UNICODE_NORM_FORM)
       …

```

(I have real trouble spelling 'normilization' consistently. Maybe an abbreviation would be helpful? 😄 ) 

In that way, someone that wants to widely use a different normalization form, or disable it altogether, would do so with a subclass, instead of needing to remember to pass the argument to every instance of the field:
```python
class NonNormalizedText(Text):
    DEFAULT_UNICODE_NORM_FORM = None
```